browser/src/control/jsdialog/Widget.MenuButton.js

```diff
@@ -114,6 +114,8 @@ function _menubuttonControl (parentContainer, data, builder) {
 				onSelect: function (event) {
 					if (event.item.uno)
 						builder.map.sendUnoCommand('.uno:' + event.item.uno);
+					else if (event.item.id === 'print-active-sheet' || event.item.id === 'print-all-sheets')
```

Comment:
please rpepare generic solution, use `map.dispatch(event.item.xxx)`

Reply:
not if (myspecial id, but detect something form JSON which will mean "use dispatch"

Reply:
eg. `if (item.type == 'action') map.dispatch(item.id)`